App.js

```diff
@@ -1,25 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import SvgComponent from "./LogoSvg";
+import LogoSvg from "./LogoSvg";
+import Quotes from "./Quotes";
+import AnimateSvg from './AnimateSvg';
+import AnimateQuotes from './AnimateQuotes';
 
 export default function App() {
-
   return (
-    <View style={styles.container}>
-      <SvgComponent />
+    <View style={styles.window}>
+      <AnimateSvg>
+        <LogoSvg />
+      </AnimateSvg>
+      <AnimateQuotes>
+        <Quotes />
+      </AnimateQuotes>
     </View>
   )
 };
 
 const styles = StyleSheet.create({
-  container: {
+  window: {
     flex: 1,
     justifyContent: 'center',
     paddingHorizontal: 20,
-    backgroundColor: 'black'
+    backgroundColor: 'black',
+    position: 'relative',
   },
```

Comment:
No text in app component requiring styling.

Reply:
There is text styling here in `Quotes.js`, FYI. Not sure where these styles should live. The top stylesheet for `color: white` might be good

Reply:
I had originally tried to use the top stylesheet to affect the text in `Quotes.js`, but the styling did not behave as expected with this application. However, when I moved the styling to the `Quotes.js` component, everything behaved as expected, so I saw this as a reason to keep the text styling at the specific lower component level.